ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/AddressIndexedSignerProvider.java

```diff
@@ -62,7 +62,11 @@ public static AddressIndexedSignerProvider create(final SignerProvider signerPro
   public Optional<Signer> getSigner(final String address) {
     final ECPublicKey publicKey = addressToPublicKeyMap.get(address.toLowerCase());
     if (publicKey == null) {
-      return Optional.empty();
+      // attempts to load config/signer via address
+      final Optional<Signer> signer = signerProvider.getSigner(address.toLowerCase());
```

Comment:
The SignerProvider ideally shouldn't know about Ethereum addresses and be generic. Perhaps could just change the signer provider to have a generic identifier instead of a public key?